daemon/network.go

```diff
@@ -426,9 +426,10 @@ func (daemon *Daemon) createNetwork(ctx context.Context, cfg *config.Config, cre
 func (daemon *Daemon) pluginRefCount(driver, capability string, mode int) {
 	var builtinDrivers []string
 
-	if capability == driverapi.NetworkPluginEndpointType {
+	switch capability {
+	case driverapi.NetworkPluginEndpointType:
 		builtinDrivers = daemon.netController.BuiltinDrivers()
-	} else if capability == ipamapi.PluginEndpointType {
+	case ipamapi.PluginEndpointType:
```

Comment:
can you add a `default:` in this switch as well? (can be an empty branch with a comment that we don't handle other types (not sure if we should have logs for those?))